### PR TITLE
feat: add restaurant links

### PR DESCRIPTION
**Pull request by callumcurtis**

### CODE REVIEW

File: packages/schema/src/atoms/index.ts [hunk 1]
```diff
@@ -77,3 +78,7 @@ export const ReviewIdSchema = z.number().int().positive();
 export const RestaurantAddressSchema = z.string().min(1).max(255);
 
 export const RestaurantDescriptionSchema = z.string().min(1).max(255);
+
+const UrlSchema = z.string().url();
+
+export const RestaurantUrlSchema = UrlSchema;
```
Comment (anormananderson) [inline packages/schema/src/atoms/index.ts:84]: I think this export should match the style of the rest of the file.
```suggestion
export const RestaurantUrlSchema = z.string().url();
```

**Review by anormananderson — DISMISSED** [review]

**Review by anormananderson — COMMENTED** [review]
I also think there should be some visual indicator to distinguish clickable restaurant titles from non-clickable ones. But I'm also not sure how to do that unintrusively.

File: packages/schema/src/atoms/index.ts (hunk 1 above)
Comment (callumcurtis) [reply packages/schema/src/atoms/index.ts:84]: Separated UrlSchema out here to reflect that RestaurantUrlSchema is just a regular URL schema - the other schemas are more specific, hence why I didn't extract anything out from them.

**Comment by callumcurtis** [comment]
> I also think there should be some visual indicator to distinguish clickable restaurant titles from non-clickable ones. But I'm also not sure how to do that unintrusively.

Worth creating a ticket for and thinking about - we agreed offline to leave as-is for now though.

**Pull request merged**